packages/webapp/public/locales/de/disease.json

```diff
@@ -1,528 +1,528 @@
 {
-  "PESTICIDE": {
-    "BORDEAUX_MIXTURE": "Bordeaux-Mischung",
-    "DIPEL": "Dipel 2X DF",
-    "NEEM_OIL": "Neem-Öl",
-    "ROUNDUP": "Roundup",
-    "SULFOCALCIUM_SYRUP": "Sulfocalcium-Sirup"
-  },
-  "group": {
-    "BACTERIA": "Bakterien",
-    "FUNGUS": "Fungus",
-    "INSECT": "Insekt",
-    "MITE": "Milbe",
-    "OTHER": "Andere",
-    "VIRUS": "Virus",
-    "WEED": "Unkraut"
-  },
   "name": {
```

Comment:
Just noticing we have these lowercase object keys -- don't love that but this is correct.